libraries/AP_PiccoloCAN/piccolo_protocol/ServoDefines.h

```diff
@@ -0,0 +1,191 @@
+// ServoDefines.h was generated by ProtoGen version 3.2.a
+
+/*
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: Oliver Walters / Currawong Engineering Pty Ltd
+ */
+
+#ifndef _SERVODEFINES_H
+#define _SERVODEFINES_H
+
+// Language target is C, C++ compilers: don't mangle us
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*!
+ * \file
+ */
+
+#include <stdbool.h>
+#include "ServoProtocol.h"
+
+/*!
+ * Servo status bits
+ */
+typedef struct
+{
+    unsigned   enabled : 1;     //!< Servo enabled status
+    ServoModes mode;            //!< Servo mode
+    unsigned   cmdReceived : 1; //!< Set if the servo has received a valid position command within the configured timeout period
+    unsigned   reservedA : 1;   //!< Reserved for future use
+    unsigned   reservedB : 1;   //!< Reserved for future use
+    unsigned   reservedC : 1;   //!< Reserved for future use
+}Servo_StatusBits_t;
```

Comment:
@peterbarker for example these status bits (and the warning and error bits below)